webrtc-stats.html

```diff
@@ -2106,6 +2087,19 @@
                   Represents the total number of consent requests sent.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>consentExpiredTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the latest valid STUN binding response expired,
+                  as defined in [[!RFC7675]] section 5.1. The time is relative to the UNIX epoch
+                  (Jan 1, 1970, UTC). If a valid STUN binding response has not been made
```

Comment:
maybe the reference to UNIX epoch can be made in the section where we define timestamps in the introduction